include/aos/common/spaceallocator/spaceallocator.hpp

```diff
@@ -114,6 +114,107 @@ class SpaceAllocatorItf {
     virtual ~SpaceAllocatorItf() = default;
 };
 
+/**
+ * Space instance.
+ */
+class Space : public SpaceItf {
```

Comment:
Let's create Space with the given space, return not 0, but some meaningful data.
Also, let's implement resize logic (simply change underlaying data).

With this changes, such a Space object will be useful for Layer/Service manager, `cause image::ImageHandler returns space object, from which Size is taken to store it in DB.